src/components/form-attachment/list.vue

```diff
@@ -32,8 +32,7 @@ except according to the terms contained in the LICENSE file.
           <sentence-separator/>
           <i18n-t keypath="moreInfo.clickHere.full">
             <template #clickHere>
-              <!-- TODO. Specify the `to` prop. -->
-              <doc-link>{{ $t('moreInfo.clickHere.clickHere') }}</doc-link>
+              <doc-link to="central-datasets/">{{ $t('moreInfo.clickHere.clickHere') }}</doc-link>
```

Comment:
The surrounding text is:

> One or more Form Attachments have filenames that match Dataset names. By default, those are linked to Datasets. For testing, you may want to upload temporary data as .csv files, then link to the Datasets once you have verified your form logic. For more information, **click here**.

I thought about linking to the "Testing Forms with Datasets" section of the page, since that seems most relevant to the second-to-last sentence. But looking at the text as a whole, I think it makes the most sense to link to the whole page.